<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>File Management</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/file.png</normaloff>images/file.png</iconset>
  </property>
  <widget class="QPushButton" name="btn_manageCenter">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>批量管理中心</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_searchCenter">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>查询中心</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_backFile">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>260</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>返回更新文件</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
